governance.md

```diff
@@ -0,0 +1,340 @@
+vote. If they plan to return to active participation soon, they will be
+given a grace period of one year. If they don’t return to active participation
+within that time period they will be removed by vote of the Council without
+further grace period. All former Council members can be considered for
+membership again at any time in the future, like any other Project Contributor.
+Retired Council members will be listed on the project website, acknowledging
+the period during which they were active in the Council.
+An inactive BDFL may also be ejected by a consensus of current members under
+these same conditions.
+
+The Council reserves the right to eject current Members, other than the BDFL,
+if they are deemed to be actively harmful to the project’s well-being, and
+attempts at communication and conflict resolution have failed.
+
+
+### Conflict of interest
+
+It is expected that the Council Members and BDFL will be employed at a wide
+range of companies, universities and non-profit organizations. Because of this,
+it is possible that Members will have conflict of interests. Such conflict of
+interests include, but are not limited to:
+
+-   Financial interests, such as investments, employment, or contracting work,
+    outside of The Project that may influence their work on The Project.
+-   Access to proprietary information of their employer that could potentially
+    leak into their work with the Project.
+-   An issue where the person privately gains an advantage from The Project
+    resources, but The Project has no gain or suffers a disadvantage.
+
+All members of the Council, BDFL included, shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused his/herself for a particular decision, they will appoint a substitute
+BDFL for that decision.
+
+### Private communications of the Council
+
+Unless specifically required, all Council discussions and activities will be
+public and done in collaboration and discussion with the Project Contributors
+and Community. The Council will have a private mailing list that will be used
+sparingly and only when a specific matter requires privacy. When private
+communications and decisions are needed, the Council will do its best to
+summarize those to the Community after eliding personal/private/sensitive
+information that should not be posted to the public internet.
+
+### Subcommittees
+
+The Council can create subcommittees that provide leadership and guidance for
+specific aspects of the project. Like the Council as a whole, subcommittees
+should conduct their business in an open and public manner unless privacy is
+specifically called for. Private subcommittee communications should happen on
+the main private mailing list of the Council unless specifically called for.
```

Comment:
What sort of information are the potential subcommittees suppose to provide to support their case? Who is allowed to be a member of these subcommittees (core, contributors, etc.)? Is there a vote for creating the subcommittees? When are subcommittees dissolved (if ever)? Are there existing subcommittees that should be listed here (e.g. CoC)? Where do the rosters for their members live? How should one contact a subcommittee about an issue?

Reply:
I don't have good answers for these (I'm mostly copying from the Jupyter doc here).  Is this something that you think we should get into?  If so, would you be interested in leading that process?

Reply:
Ping @jakirkham

Reply:
Was waiting to see if other members thoughts on these questions before voicing my own.

My suggestion would be to drop the subcommittees section. It seems a bit too heavyweight for our needs today.

Reply:
Taking a look at this now.  I suspect that the original intent was to provide a structure for the NumFOCUS subcommittee in the next section.

Reply:
Looks like this was changed below. Marking as resolved. Please reopen if not.

Reply:
This being for numfocus makes sense.

Reply:
At least in conda-forge we use subcommittees to accelerate decision making on some focused issue by a small group of interested individuals that the larger group may not be tracking in detail. I'm not entirely sure how Jupyter uses them (outside of the NumFOCUS subcommittee).

Reply:
I've pushed some text to fill out some answers to the questions above. This follows roughly how subcommittees are used in conda-forge (e.g. to defer decisions to a select group of experts on an issue). Though am not attached to this model if there are other use cases we'd like to cover. Please take a look and share your thoughts.